### ISSUE TITLE
Unable to push on corporate proxy 

### ISSUE THREAD

**Issue by alik604** [desktop/desktop#9741]
#### Support usage with corporate proxy.
I can pull data, but not push. This is because I can't login. I get the following error when I attempt _(see screenshot)_ 

I tried `git config --global http.proxy http://USER@{X.X.X.X}:8080`, that let me be able to do some action ( which I can not recall) 

_I could avoid this issue if I exited the corporate VPN. I doubt I am the only one this this pet-peeve. I there should be a proxy field. as I did for PIP `pip install --proxy=https://USER@X.X.X.X:8080 numpy
`
> Edit: if I exit the corporate VPN and log-in, I get the following error when I attempt to push.
perhaps this is due to me using `git config --global http.proxy http://USER@{X.X.X.X}:8080`
![image](https://user-images.githubusercontent.com/29245410/81462693-409f5980-9169-11ea-9881-1db3863f46ba.png)

### Version & OS

Windows 10. 
Latest build of Github desktop 2.4.3

### Screenshots

![image](https://user-images.githubusercontent.com/29245410/81462398-10ef5200-9167-11ea-9b57-453d745d3d2c.png)

**Comment by outofambit** [comment]
hi @alik604 was the screenshot attached what you saw before you ran any manual git cli proxy config commands? what happened when you filled out the username + password in the "Authentication failed" popup?

**Comment by alik604** [comment]
Hi @outofambit 
The screenshot is from after the git cli proxy config. 

If I recall correctly, after I entered my username & password in the "Authentication failed" popup, it would try, fail, and show the popup again. 

I just tried to push to my repo, trying to recreate the message. However, the push worked as successful. This was the first time I opened GitHub this week (this issue was opened on Friday night). 

_I guess restarting fixed the issue_
